programs/cleos/main.cpp

```diff
@@ -2299,9 +2299,9 @@ void get_account( const string& accountName, const string& coresym, bool json_fo
          return ss.str();
       };
 
-
+      const std::string net_current_usage_str = "    ( " + (res.net_limit.current_used.valid() ? to_pretty_net(*res.net_limit.current_used) : "out of date") + " )";
       std::cout << std::fixed << setprecision(3);
-      std::cout << indent << std::left << std::setw(11) << "used:"      << std::right << std::setw(18) << to_pretty_net( res.net_limit.used ) << "\n";
+      std::cout << indent << std::left << std::setw(11) << "used:"      << std::right << std::setw(18) << to_pretty_net( res.net_limit.used ) << net_current_usage_str << "\n";
```

Comment:
It should not print the `used` value if `current_used` is available. Only one value should be printed. If that value is `used` (because `current_used` does not exist) then you add the `(out of data)` at the end of the line.